spring-boot-project/spring-boot-tools/spring-boot-configuration-processor/src/json-shade/java/org/springframework/boot/configurationprocessor/json/JSONArray.java

```diff
@@ -65,8 +65,8 @@ public JSONArray() {
 	public JSONArray(Collection copyFrom) {
 		this();
 		if (copyFrom != null) {
-			for (Iterator it = copyFrom.iterator(); it.hasNext();) {
-				put(JSONObject.wrap(it.next()));
+			for (Object it : copyFrom) {
```

Comment:
When merging, I think we should leave this as-is so that it matches the original source.

Reply:
Ok, thank you for your letter.

Reply:
> it matches the original source

@wilkinsona I'm sorry but I don't quite understand, what's the difference of these two versions?

Reply:
There's no functional difference, but the source code is different. This code is a copy of code authored elsewhere and we don't want to diverge from its original form.

Reply:
OK, thanks for clarification! :)